entra-id-quarkus/src/main/java/no/kantega/ProfilePage.java

```diff
@@ -0,0 +1,58 @@
+package no.kantega;
```

Comment:
Hold on a minute here. Do we have permission to take this code from https://en.wikipedia.org/wiki/Kantega ? I know this is a sample, but we need to ensure the code has a license that allows us to use it like this.

Reply:
Good catch. The sample code here does reference https://github.com/kantega/quarkus-ad which has a MIT license, so we can use code like this. Moreover, I rename the groupId/package name to `com.example` to reflect that the code sample is based on the auto-generated code base with:

```
mvn io.quarkus.platform:quarkus-maven-plugin:3.11.3:create \
    -DprojectGroupId=com.example \
    -DprojectArtifactId=quarkus-ad \
    -Dextensions='oidc,rest-qute,rest-jackson' \
    -DjavaVersion="21"
```